app/models/item.rb

```diff
@@ -0,0 +1,10 @@
+class Item
+  attr_reader :name, :id, :description, :unit_price
+
+  def initialize(item)
+    @name = item[:attributes][:name]
+    @id = item[:id]
+    @description = item[:attributes][:description]
+    @unit_price = item[:attributes][:unit_price]
+  end
+end
```

Comment:
This goes for the merchant.rb poro as well, but be really careful naming an attribute "id", since all poros will automatically have a generated id on them. This could cause issues later on when you're trying to access the id that you manually stored here.